scripts/publish-mavencentral.gradle

```diff
@@ -0,0 +1,130 @@
+            groupId PUBLISH_GROUP_ID
+            artifactId PUBLISH_ARTIFACT_ID
+            version PUBLISH_VERSION
+
+            // Two artifacts, the `aar` (or `jar`) and the sources
+            if (project.plugins.findPlugin("com.android.library")) {
+                artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
+            } else {
+                artifact("$buildDir/libs/${project.getName()}-${version}.jar")
+            }
+            artifact androidSourcesJar
+
+            pom {
+                name = PUBLISH_ARTIFACT_ID
+                description = 'Android client for push notifications from cloudfiveapp.com'
+                url = 'https://github.com/cloudfive/push-android'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'brian'
+                        name = 'Brian Samson'
+                        email = 'brian@cloudfiveapp.com'
+                    }
+
+                    developer {
+                        id = 'brett'
+                        name = 'Brett Samson'
+                    }
+
+                    developer {
+                        id = 'alex'
+                        name = 'Alex Petitjean'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:https://github.com/cloudfive/push-android.git'
+                    developerConnection = 'scm:git:https://github.com/cloudfive/push-android.git'
+                    url = 'https://github.com/cloudfive/push-android.git'
+                }
+                // A slightly hacky fix so that your POM will include any transitive dependencies
+                // that your library builds upon
+                withXml {
+                    def dependenciesNode = asNode().appendNode('dependencies')
+
+                    project.configurations.implementation.allDependencies.each {
+                        def dependencyNode = dependenciesNode.appendNode('dependency')
+                        dependencyNode.appendNode('groupId', it.group)
+                        dependencyNode.appendNode('artifactId', it.name)
+                        dependencyNode.appendNode('version', it.version)
+                    }
+                }
+            }
+        }
+    }
+    // The repository to publish to, Sonatype/MavenCentral
+    repositories {
+        maven {
+            // This is an arbitrary name, you may also use "mavencentral" or
+            // any other name that's descriptive for you
+            name = "sonatype"
+            url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+//            url = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
```

Comment:
yea this is the old repo, the new `s01` is where everything is now. idk why the move